benchmark/variable_benchmark.cpp

```diff
@@ -66,14 +66,18 @@ template <class Gen> static void BM_Variable_copy(benchmark::State &state) {
   auto [var, size] = Gen()(axisLength);
   for (auto _ : state) {
     Variable copy(var);
+    state.PauseTiming();
+    copy = Variable();
```

Comment:
Is this preventing compiler optimisation or caching?

Reply:
Neither. It should ensure that we measure only the time for the actual copy. Without this the deallocation is also timed. Resetting the variable to something empty outside the timed section should avoid this.